src/dtos/unit-control.dto.ts

```diff
@@ -137,7 +137,7 @@ export class UnitControlBaseDTO {
   @IsOptional()
   @IsInRange(0, 1, {
```

Comment:
A Range Validator is not really appropriate for string "indicator" values of 0 and 1.  Maybe an enum or regex validator.

Reply:
updated the validation using @MatchesRegex.